aiohttp/web_response.py

```diff
@@ -697,6 +697,8 @@ def body(
     def text(self) -> Optional[str]:
         if self._body is None:
             return None
+        if isinstance(self._body, Payload):
```

Comment:
Could you explain this bit? I expected the fix to be in the exception initializer.